src/styles/watch.scss

```diff
@@ -0,0 +1,56 @@
+.stopwatch {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  margin: 0 auto;
+  background-color: aquamarine;
```

Comment:
let's add background for body, because now it is:
![image](https://github.com/mate-academy/layout_stop-watch/assets/102289654/ce807fd1-f2b5-4db5-9133-29a91775d0c5)